freebsd/FreeBSDProcess.h

```diff
@@ -13,12 +13,23 @@ in the source distribution for its full text.
 #include "Process.h"
 #include "Machine.h"
 
+typedef enum {
+   SCHEDCLASS_UNKNOWN = 0,
+
+   SCHEDCLASS_ITHD, /* interrupt thread */
```

Comment:
Not the most expressive name. Unless forced by some reference documentation I'd prefer something like `SCHEDCLASS_IRQTHREAD`.

Reply:
@BenBE The original constant is named `PRI_ITHD`, but I do agree it can be named better here in htop. My personal suggestion is `SCHEDCLASS_INTR_THREAD`.